labeler.js

```diff
@@ -167,8 +175,23 @@ async function main() {
     if (cli.flags.path) helper.labelsPath() // Return labels.json path
     if (cli.flags.resetLabelsFile) await helper.resetLabelsFile(cli) // Reset labels.json file
     if (cli.flags.emptyLabelsFile) await helper.emptyLabelsFile(cli) // Delete all labels from labels.json
-    if (cli.flags.deleteAllLabels) await helper.deleteAllLabels(token, owner, host, repository, cli) // Delete all labels from repository
-    if (cli.flags.uploadLabels) await helper.uploadLabels(token, owner, host, repository, cli) // Upload custom labels to repository
+
+    // This will delete and/or upload all labels to every repository under the owner organization in GHE
+    // Currently only handles GHE instances, but could probably be adapted for a GitHub user
+    if (cli.flags.bulkUpdate) {
+        let exit = false;
```

Comment:
Personal preference: I do not use `;` to end a line, unless absolutely necessary.
```suggestion
        let exit = false
```